### ISSUE TITLE
Version Update ignores web dir setting for new web files 

### ISSUE THREAD

**Issue by TechieAndroid** [webbukkit/dynmap#3069]
![image](https://user-images.githubusercontent.com/17918163/89371251-75f40e80-d6a8-11ea-91e4-da1763aa6df5.png)

After updating to the latest version of Dynmap I got this strange error. I tried reloading without the cache but it didn't affect it.
I looked all over the wiki and searched online for an answer but found nothing.

A few people responded to my Reddit post and recommended updating the web dir with the "new files" but I didn't have anything new generate to replace them with.

All in all, it was a very weird error and I think there should be more information on it if this is more common and if there is a case where my solution doesn't apply.

My solution:
edit version.js to reflect the latest version of Dynmap.

Is this a perfect solution? No. And I'm certain that there's more to it than what I did but since it's working for now I'm alright with leaving it.

Feel free to close this issue if you don't think it's important.

**Comment by FedUpWith-Tech** [comment]
This is a very strange bug, I've seen it three times now but haven't been able to replicate it myself. All of them were upgrades to beta-2 from various versions. 

What version did you upgrade from? The web files haven't changed in a longggg time so its strange that its mad with this version.

**Comment by TechieAndroid** [comment]
Yes that was the same version that I update to if I'm not mistaken.

So I just updated again and the same thing happened but I believe I've figured the problem out.

What's happening is dynmap config is not being respected when you update the jar file. So let's say you have the web dir in a Apache server and the config is pointing to said location. When you update the dynmap jar it doesn't care about that config and dumps the updated web files in the plugins dir instead.

This is really messed up and can lead to users mistakenly deleting the _new_ webfiles generated in the plugins dir, thinking they're old and don't need them, since they (the updated files) are supposed to be going to the location specified in the config.

Hopefully that makes sense. I believe that's what happened to me before. I accidentally deleted the web folder that was supposed to be in the Apache dir but was actually sitting in the plugins dir after updating.

It's a simple mistake anyone can make, especially since there's no documentation on updating dynmap or those web dir files and how it doesn't respect alternate web dir paths when updated.
